packages/discord.js/src/structures/BaseInteraction.js

```diff
@@ -270,16 +270,18 @@ class BaseInteraction extends Base {
     return this.type === InteractionType.MessageComponent && this.componentType === ComponentType.Button;
   }
 
-  // TODO: Get rid of this in the next major
   /**
    * Indicates whether this interaction is a {@link StringSelectMenuInteraction}.
+   * @method isSelectMenu
+   * @memberof BaseInteraction
+   * @instance
    * @returns {boolean}
-   *
    * @deprecated Use {@link BaseInteraction#isStringSelectMenu} instead.
    */
-  isSelectMenu() {
-    return this.isStringSelectMenu();
-  }
+  isSelectMenu = deprecate(
```

Comment:
This is called on every single instance, which resets the deprecation warning state.

Reply:
That's unfortunate. Undid this with ea9ca1ea528a0cf197baca8ced5794b39ecbba72.